_includes/open_graph.html

```diff
@@ -0,0 +1,25 @@
+<meta property="og:url" content="{{ site.url }}{{ page.url }}" />
+<meta property="og:type" content="article" />
+<meta property="og:title" content="{{ page.title | default: site.title }}" />
+{% if page.summary %}
+  {% assign description = page.summary %}
+{% elsif page.questions %}
+  {% assign description = page.questions | join: " " %}
+{% elsif page.keypoints %}
+  {% assign description = page.keypoints | join: " " %}
+{% elsif include.description %}
+  {% assign description = include.description %}
+{% else %}
```

Comment:
Individual episode pages will have three things in their YAML header: objectives, questions, and keypoints. I would suggest if the impetus for prioritizing questions is the fact that they have question marks delimiting them, then we can delimit the list items with em-dashes. 

```suggestion
{% if page.summary %}
  {% assign description = page.summary %}
{% elsif page.objectives %}
  {% assign description = page.objectives | join: "&mdash;" %}
{% elsif page.questions %}
  {% assign description = page.questions | join: "&mdash;" %}
{% elsif page.keypoints %}
  {% assign description = page.keypoints | join: "&mdash;" %}
{% elsif include.description %}
  {% assign description = include.description %}
{% else %}
```

Reply:
I think the emdash would work, yes, provided the content is HTML-decoded, which I'm not sure about. If it's not, which is the basis I was working on, joining with `" - "` or `" --- "` would work.